runtime/src/main/java/org/corfudb/runtime/view/stream/ReplexStreamView.java

```diff
@@ -49,6 +50,46 @@ public ReplexStreamView(final CorfuRuntime runtime,
 
     /** {@inheritDoc}
      *
+     * This is problematic in Replex since we need to translate
```

Comment:
I tend to disagree. This branch changes the stream API, so it effects the Replex implementation significantly. We are unable and don't wish to properly test the changes to the Replex option at this time.